src/Components/SuperAdmin/Admins/Form/index.jsx

```diff
@@ -179,7 +188,7 @@ const Form = () => {
             <Button text={id ? 'Update' : 'Add'} type="submit" info={'submit'} />
             <div className={styles.buttonsAdmin}>
               <Button text="Back" type="cancel" clickAction={handleCancel} />
-              <Button type={'cancel'} clickAction={handleReset} info={'reset'} text={'Reset'} />
+              <Button type={'cancel'} onClick={handleReset} info={'reset'} text={'Reset'} />
```

Comment:
You should change the onClick={handleReset} for clickAction={handleReset} to correct the implementation of the shared Input.